CONTRIBUTING.md

```diff
@@ -17,3 +17,14 @@ place. Ideas for new extensions and questions about how to correctly implement
 or consume an API that adheres to the JSON:API specification belong here.
 
 [about]: http://jsonapi.org/about
+
+## JSON Schema
+If you would like to update the JSON schemas, you must first install the node.js dependencies :
+```
+npm install
+```
+Then you can run the schema validator by running the following command :
+```
+node ./_schemas/scripts/validator.js
+```
```

Comment:
Is it also possible to validate one test file using [`ajv-cli`](https://ajv.js.org/packages/ajv-cli.html)? I tried but wasn't able to figgure out correct options. Always ended up with this error:

```
$ ajv test --spec=draft2020 -s _schemas/1.0/schema_create_resource.json -d _schemas/1.0/tests/request/resource/create/valid/post_resource.json --valid
schema _schemas/1.0/schema_create_resource.json is invalid
error: no schema with key or ref "http://json-schema.org/draft-07/schema#"
```

Reply:
You can not test a single file using ajv-cli for 2 reasons : 

- the json:api schema uses formats (uri, in particular) which are not included by default in the ajv validator. We need to add the ajv-format dependency to use them.
- the schemas to validate request use refenrencies to the main schema and when you use the `-s` option of the ajv-cli, you can specify only one schema. So the ajv validator doesn't know where to find the referencies...

Also, in your example, the `--spec` option must have `draft7 `as its value ;-)

I'll try to add this possibility to the script...

Reply:
I updated the script : now we have 2 new options : 

- `-v version` : to validate a specific version of the spec
- `-f file_path` : to validate only one test file

These 2 options can be mixed.

So, now, you can run : 
`node ./_schemas/scripts/validator.js -v 1.1 -f _schemas/1.1/tests/request/resource/create/valid/post_resource_with_lid_member.json`
or
`npm run test-schema -v 1.1 -f _schemas/1.1/tests/request/resource/create/valid/post_resource_with_lid_member.json`